FrontEnd/src/components/SearchPage/search_field/companies/Companies.js

```diff
@@ -0,0 +1,106 @@
+import { Link } from 'react-router-dom';
+import wish_list_checklist from './img/wish_list_checklist.svg';
+import wish_list_checklist_added from './img/wish_list_checklist_added.svg';
+import './CompaniesCards.css';
+
+const MainCompanies = ({ companyData, isAuthorized }) => {
+  const currentDate = new Date();
+  const currentYear = currentDate.getFullYear();
+
+  // those variables we would use for axios to get data from beckend
+  // get saved list code here
+  const usersSavedList = [];
+
+  // add company to saved list code here
+  // const addToSavedList = (profile_id) => {};
+
+  // del company from saved list code here
+  // const delFromSavedList = (profile_id) => {};
+
+  return (
+    <div className="product-card">
+      <div className="product-card__block">
+        <div className="product-card__image-frame">
+          <img
+            className="product-card__image"
+            src={companyData.banner_image}
+            alt={companyData.name}
+          />
+        </div>
+        <div className="product-card__text-block">
+          <div className="product-card__text-block__header">
+            <div className="product-card__category-text align_items_left">
+              {companyData.categories &&
+                companyData.categories
+                  .map((category) => category.name)
+                  .join(' ')}
+            </div>
+            <div className="product-card__name-text align_items_left">
```

Comment:
Do we have a product card component? If no its better to move it to the separate component

Reply:
I've renamed Companies.js to CompanyCard. Because it ctreates company card, not a list of companies. Is that smth you've meant?

Reply:
If it creates CompanyCard I think its ok.